bin/govuk-docker

```diff
@@ -32,6 +34,20 @@ function check_stack {
 }
 
 case $COMMAND in
+  "setup")
+    for GOVUK_DOCKER_SERVICE in "$@"; do
+      check_service
+    done
+    set -x
+    cd $GOVUK_DOCKER_DIR
+    for GOVUK_DOCKER_SERVICE in "$@"; do
+      make $GOVUK_DOCKER_SERVICE
+    done;;
+  "setup-this")
+    check_service
+    set -x
+    cd $GOVUK_DOCKER_DIR
+    make $SERVICE;;
```

Comment:
Is that a typo here?

Reply:
Where?

Reply:
We seem to have two semicolons - is that necessary?

Reply:
Yes, it's how to end a `case` branch (the other branches do this too)